TestFoundation/TestURL.swift

```diff
@@ -396,6 +396,8 @@ class TestURL : XCTestCase {
         let lengthOfRelativePath = Int(strlen(TestURL.gFileDoesNotExistName))
         let relativePath = fileSystemRep.advanced(by: Int(TestURL.gRelativeOffsetFromBaseCurrentWorkingDirectory))
         XCTAssertTrue(strncmp(TestURL.gFileDoesNotExistName, relativePath, lengthOfRelativePath) == 0, "fileSystemRepresentation of file path is wrong")
+
+        FileManager.default.changeCurrentDirectoryPath(TestURL.gSavedPath)
```

Comment:
Not necessary since it's reset in the next test, but keeps these tests independent so if someone shuffles the order of the tests someday, that won't matter.